### PR TITLE
Add ability to kill from port

### LINKED ISSUE #19: Search by port
It'd be great if it's possible to also search by a port being open by a process

### PR DESCRIPTION
**Pull request by kevva**
Fixes #19.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Looks really good!

**Comment by sindresorhus** [comment]
When I try to kill a port that is *not* in use, I get this:

```
❯ fkill :8000
? Error killing process. Would you like to use the force? Yes
Error: Couldn't find a process with port `8000`
    at getPort (/Users/sindresorhus/dev/oss/fkill/node_modules/pid-from-port/index.js:21:9)
    at getList.then.list (/Users/sindresorhus/dev/oss/fkill/node_modules/pid-from-port/index.js:41:32)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

It should not suggest force killing.

**Comment by kevva** [comment]
> It should not suggest force killing.

Sounds reasonable. Fixed.

File: cli.js [hunk 1]
```diff
@@ -113,6 +139,13 @@ if (cli.input.length === 0) {
 	const promise = fkill(cli.input, Object.assign(cli.flags, {ignoreCase: true}));
 
 	if (!cli.flags.force) {
-		promise.catch(() => handleFkillError(cli.input));
+		promise.catch(err => {
+			if (/Couldn't find a process with port/.test(err.message)) {
```
Comment (sindresorhus) [inline cli.js]: I think you meant `if (/Process doesn't exist/.test(err.message)) {` here ;)

Comment (kevva) [reply cli.js]: I can add that too, but in this case we get the `Couldn't find a process with port` from the `pid-from-port` module.

Comment (sindresorhus) [reply cli.js]: Weird, it works now. Not sure what changed...

**Pull request merged**